integrations/crewai/README.md

```diff
@@ -0,0 +1,211 @@
+# Using Unity Catalog AI with CrewAI 
+
+You can use functions defined within Unity Catalog (UC) directly as tools within [CrewAI](https://www.crewai.com/) with this package.
+
+## Installation
+
+### From PyPI
+
+```sh
+pip install ucai-crewai
+```
+
+### From source
+
+To get started with the latest version, you can directly install this package from source via:
+
+<!-- TODO: update this to the actual path where the repo's main branch will live -->
+```sh
+%pip install git+https://github.com/michael-berk/unitycatalog-ai.git@michaelberk/crewai-tool-integration#subdirectory=integrations/crewai
```

Comment:
```suggestion
pip install ucai-crewai
```

Reply:
The namespace is going to be changed as well, but we'll handle that in the unitycatalog repo :)